.circleci/config.yml

```diff
@@ -61,7 +61,6 @@ jobs:
             mamba install python=3.10 julia r-base rpy2 \
```

Comment:
why remove Julia instead of pinning the version here?

Reply:
it uses Julia for one of the example

Reply:
Julia is installed here and 2 lines below too, so I removed the second instance

Reply:
The second one is for `pyjulia`, which is necessary to get the curve with julia in the example with R and Julia.
(the trick is: `conda install julia` installs the julia interpreter while `pip install julia` install the interface to call julia from python, both are needed).